ingestclient/utils/filesystem.py

```diff
@@ -143,7 +146,29 @@ def get_file(self, path):
             (io.BufferedReader): A file handle for the specified file
         """
         output = six.BytesIO()
-        self.bucket.download_fileobj(path, output)
+
+        # Allows for wildcards (Involves data crawl, computationally expensive)
+        if '*' in path:
+            segments = path.split('*')
+            prefix = segments[0]
+            suffix = segments[1]
+            ## List objects only goes up to about 1000 objects, paginator acts as bypass
+            pageresponse = self.paginator.paginate(Bucket=self.bucketname, Prefix=prefix)
+            for pageobject in pageresponse:
+                try: 
+                    for key in pageobject['Contents']:
```

Comment:
As written, this will search the entire bucket and return data from the _last_ matching filename. I'm not sure what use case you're trying to meet. I'd think you'd either want to return the first match or return a list with the data of all the matches.

Reply:
Hmmm..... 
Will have to look more into how to do this with boto3.